docs/appendices/release-notes/5.6.5.rst

```diff
@@ -47,6 +47,10 @@ See the :ref:`version_5.6.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that could prevent a :ref:`sql-restore-snapshot` from
```

Comment:
I'll follow up on the vale failures later. `sql` is part of our custom vocabulary, so this shouldn't fail.

Seems related to the version update, although I'm not sure why. According to https://vale.sh/docs/topics/vocab/ I think I updated the path correctly.